scripts/linters/custom_eslint_checks/run_custom_eslint_tests.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script for running tests for custom eslint checks."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import subprocess
+import sys
+
+import python_utils
+
+from scripts import common
+
+
+def main():
+    """Run the tests."""
+    node_path = os.path.join(common.NODE_PATH, 'bin', 'node')
+    mocha_path = os.path.join('node_modules', 'mocha', 'bin', 'mocha')
+    filepath = 'scripts/linters/custom_eslint_checks/tests/lib/rules/'
```

Comment:
>  we do not be able to know where the test fail

So how the user will fix the test issues in eslint_tests?

Reply:
This is the case if we run this eslint tests with other python unit tests, if we run it separately as we are doing right now, it will show all errors, you can check the GithubCLI for example.

Reply:
But why it doesn't show when running with other tests?

Reply:
Unit tests does not show the output of a test, they just show the status of pass or fail.

FYI, this is a testing mechanism in itself, that's why we needed it separately. I did what you suggested at my first try, but it doesn't work there. It just showed tests failed but not which one, because this is a completely different tests suite, it is cannot be worked as part of our current unittests.